GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -141,6 +143,45 @@ public override void IsServiceInstalledAndRunning(string name, out bool installe
             running = installed && gvfsService.IsRunning;
         }
 
+        public override bool TryCopyPanicLogs(string copyToDir, out string error)
+        {
+            error = null;
+            try
+            {
+                if (!Directory.Exists(DiagnosticReportsDirectory))
+                {
+                    return true;
+                }
+
+                string copyToPanicDir = Path.Combine(copyToDir, ProjFSKext.DriverLogDirectory, "panic_logs");
+                Directory.CreateDirectory(copyToPanicDir);
+
+                foreach (string filePath in Directory.GetFiles(DiagnosticReportsDirectory, PanicFileNamePattern))
+                {
+                    if (File.ReadAllText(filePath).Contains(ProjFSKext.DriverName))

```

Comment:
If you move the `File.ReadAllText` inside the inner `try` than a failure to read one panic won't stop us from checking the rest.